web-design/02_typography.md

```diff
@@ -0,0 +1,85 @@
+
+
+## 📋 Prerequisites
+- [ ] [Introduction to web design](./01_web-design-concepts.md).
+
+## 🎯 Learning Objectives
+- [ ] Understand the principles of typography and its role in visual communication.
+- [ ] Learn how to select appropriate typefaces for different design contexts.
+- [ ] Explore techniques for effective typography hierarchy and readability.
+- [ ] Gain knowledge of typographic terminology and best practices for spacing and alignment.
+
+## 📹 Video
+
+- Please watch the following videos:
+    <br>
+    **The Ultimate Guide to Typography**
+    - A comprehensive resource that covers the art and technique of arranging type to make written language legible, readable, and visually appealing. It explores the principles of typography, the selection of appropriate typefaces for different design contexts, techniques for effective typography hierarchy and readability, and best practices for spacing and alignment. The guide also delves into typographic terminology and provides insights on how to enhance the appearance and readability of text using color, images, and other visual elements.
+    <br>
+    - <a href="https://www.youtube.com/watch?v=yAuUDyUC-GM"
+    target="_blank">The Ultimate Guide to Typography</a>
+
+    [![Video](https://img.youtube.com/vi/yAuUDyUC-GM/0.jpg)](https://www.youtube.com/watch?v=yAuUDyUC-GM)
+    <br>
+
+    **How to choose fonts**
+    - Choosing fonts is an important aspect of design and it can greatly influence the perception of your content. Here are some steps to guide you:
+    Understand the Brand or Message: The font should align with the personality of the brand or the message you want to convey. For example, a playful font might not be suitable for a formal business document.
+    Consider Readability: The font should be easy to read. This is especially important for large blocks of text.
+    Consider the Medium: The font should be suitable for the medium where it will be used. For example, some fonts look good on print but not on screens, and vice versa.
+    Limit the Number of Fonts: Using too many different fonts can make the design look cluttered. A good rule of thumb is to use no more than 2-3 different fonts.
+    Test Different Options: Try out different fonts and see which one works best. Consider factors like size, spacing, and color.
+    Use Font Pairing Tools: There are online tools that can suggest fonts that go well together.Remember, the goal is to choose a font that enhances your message and makes it easy for the audience to understand.
+
+    <a href="https://www.youtube.com/watch?v=WTUM2Y11oJo" target="_blank">How to choose fonts</a>
+
+    [![Video](https://img.youtube.com/vi/WTUM2Y11oJo/0.jpg)](https://www.youtube.com/watch?v=WTUM2Y11oJo)
```

Comment:
Change this to the readme file link tags as it will be changed to the folder-based direction:
```
[How to choose fonts](here provide the link to the video for now YouTube later the folder link)
```
Change this in all of your files and video links.